File: easy/find_content_children.go
package easy

import (
	"sort"
)

func FindContentChildren(children []int, cookies []int) int {
	i := 0
	j := 0
	sort.Ints(children)
	sort.Ints(cookies)
	for i < len(children) && j < len(cookies) {
		if cookies[j] >= children[i] {
			i += 1
		}
		j += 1
	}
	return i
}

File: easy/find_content_children.py
def find_content_children(children: [int], cookies: [int]) -> int:
    if not children or not cookies:
        return 0
    children.sort()
    cookies.sort()
    i, j = 0, 0
    while i < len(children) and j < len(cookies):
        if cookies[j] >= children[i]:
            i += 1
        j += 1
    return i
